src/xmipp/libraries/reconstruction/ctf_group.cpp

```diff
@@ -398,7 +398,8 @@ bool ProgCtfGroup::isIsotropic(CTFDescription &ctf)
     cosp = COSD(ctf.azimuthal_angle);
     sinp = SIND(ctf.azimuthal_angle);
 
-    for (double digres = 0; digres < resol_error; digres+= 0.001)
+    double digres = 0;
```

Comment:
why do you define `digres` outside of the loop? (similar issue in multiple files )